asab/web/tenant/service.py

```diff
@@ -1,45 +1,61 @@
-import asab
+import aiohttp
+
+from ...abc.service import Service
+from ...config import Config
+
 from .tenant import Tenant
 
-asab.Config.add_defaults({
+# "url" is used to periodically refresh tenants from, expecting "_id" inside a JSON structure,
+# which is compatible with SeaCat Auth product
+Config.add_defaults({
 	'tenants': {
 		'ids': '',
+		'url': '',  # f. e. http://seacat-auth:8080/tenant
 	}
 })
 
 
-class TenantService(asab.Service):
+class TenantService(Service):
 
 	def __init__(self, app, service_name="asab.TenantService"):
 		super().__init__(app, service_name)
 		self.App = app
+		self.TenantWebHandler = None
 		self.Tenants = {}
-		self.TenantIds = asab.Config['tenants']['ids']
-		self.TenantIds = self.TenantIds.split(',')
 
+		# Load tenants from configuration
+		self.TenantIds = Config['tenants']['ids']
+		self.TenantIds = self.TenantIds.split(',')
 		for tenant_id in self.TenantIds:
-
 			section = 'tenant:params:{}'.format(tenant_id)
-
-			if asab.Config.has_section(section):
-				params = dict(asab.Config.items(section))
+			if Config.has_section(section):
+				params = dict(Config.items(section))
 				self.Tenants[tenant_id] = Tenant(tenant_id, params)
 			else:
 				self.Tenants[tenant_id] = Tenant(tenant_id)
 
+		# Load tenants from URL
+		self.TenantUrl = Config["tenants"]["url"]
+		if len(self.TenantUrl) > 0:
+			app.PubSub.subscribe("Application.tick/10!", self._update_tenants)
```

Comment:
Once per 5 minutes is ok, first run needs to go faster. 
Also, websocket persistent API should be added to seacat auth to feed these changes in realtime (eventually), please leave TODO here.